projects/wave-core-new/src/lib/datatable/table/table.component.ts

```diff
@@ -43,6 +44,7 @@ export class DataTableComponent implements OnInit, AfterViewInit, OnDestroy, OnC
     dataSource = new FeatureDataSource();
     displayedColumns: Array<string> = [];
     featureColumns: Array<string> = [];
+    colTypes: Array<string> = [];
```

Comment:
we have a datatype for vector columns which you should use here instead of `string` : https://github.com/geo-engine/geoengine-ui/blob/c86a273a90803303c523a8161b27a22eceed5709/projects/wave-core-new/src/lib/operators/datatype.model.ts#L318

Reply:
a61e61b